nilearn/_utils/niimg_conversions.py

```diff
@@ -176,6 +179,17 @@ def check_niimg(niimg, ensure_ndim=None, atleast_4d=False, dtype=None,
         data will be converted to int32 if dtype is discrete and float32 if it
         is continuous.
 
+    return_iterator: boolean, optional
+        Returns an iterator on the content of the niimg file input
+
+    wildcards: boolean, optional
```

Comment:
It would be nice if you refer some where giving reference to small code snippet to here. If you think its good, it is located in ```doc/manipulating_visualizing/manipulating_images.rst``` file. It would be more clear enough.